packages/shared-components/src/formio/components/core/checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,63 @@
+import { Checkbox as AkselCheckbox, CheckboxGroup } from '@navikt/ds-react';
+import BaseComponent from '../../base/BaseComponent';
+import checkboxBuilder from './Checkbox.builder';
+import checkboxForm from './Checkbox.form';
+
+class Checkbox extends BaseComponent {
+  static schema() {
+    return BaseComponent.schema({
+      label: 'Avkryssingsboks',
+      type: 'navCheckbox',
+      key: 'Avkryssingsboks',
+    });
+  }
+
+  static editForm() {
+    return checkboxForm();
+  }
+
+  static get builderInfo() {
+    return checkboxBuilder();
+  }
+
+  override setValue(value?: boolean): void {
+    // Set initial value to false if it's not required
+    if (value === null && this.component?.validate?.required === false) {
+      super.updateValue(false, { modified: true });
+    } else {
+      super.setValue(value);
+    }
+  }
+
+  changeHandler(selectedCheckboxes: string[], opts: { modified: boolean }) {
+    // There is always either 0 or 1 checkbox selected
+    const isChecked = selectedCheckboxes.length > 0;
+    super.updateValue(isChecked, opts);
+    this.rerender();
```

Comment:
Dobbeltsjekk om denne trengs

Reply:
Error og conditional

Reply:
Må ha rerender for å rendre checkboxen etter trykk